addons/crm_claim/migrations/9.0.1.0/pre-migration.py

```diff
@@ -17,7 +17,7 @@
 ]
 
 
-@openupgrade.migrate()
-def migrate(cr, version):
-    openupgrade.rename_columns(cr, column_renames)
-    openupgrade.rename_tables(cr, table_renames)
+@openupgrade.migrate(use_env=True)
```

Comment:
You're still modifying this script. Please restore it as it was for making the diff smaller.